components/cam/src/physics/crm/sam/accelerate_crm.F90

```diff
@@ -20,11 +20,10 @@
 ! -----------------------------------------------------------------------------
 module accelerate_crm_mod
     use grid, only: nx, ny
-    use shr_kind_mod, only: r8=>shr_kind_r8
     use params, only: asyncid, rc=>crm_rknd
-
     implicit none
     public
+    integer, parameter :: r8 = selected_real_kind(13)
```

Comment:
I still think these type defs should be in a centralized place, maybe in `params` if not adopted from `shr_kind_mod`?

Reply:
Yes use the kind modifiers provided by shr_kind_mod.

Reply:
Ben, this should be resolved in the latest changes.